bootstrap-salt.sh

```diff
@@ -1874,8 +1894,13 @@ __apt_get_upgrade_noinput() {
 __apt_key_fetch() {
     url=$1
 
-    # shellcheck disable=SC2086
-    __wait_for_apt apt-key adv ${_GPG_ARGS} --fetch-keys "$url"; return $?
+    tempfile="$(__temp_gpg_pub)" || return 1
```

Comment:
One small thing to finish this. The return statement becomes superfluous, because the function will be executed in a subshell. Result will always be `0` for a variable assingment. But later lines would catch errors anyway.